InfluxDB.Client.AspNetCore/InfluxDB.Client.AspNetCore.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
```

Comment:
Please add a detail information about package, something like in `Client.csproj`:

```xml
<Description>...</Description>
<Authors>influxdb-client-csharp Contributors</Authors>
<AssemblyName>InfluxDB.Client.AspNetCore</AssemblyName>
<VersionPrefix>1.16.0</VersionPrefix>
<VersionSuffix>dev</VersionSuffix>

<PackageId>InfluxDB.Client.AspNetCore</PackageId>
<PackageTags>influxdata;timeseries;flux;influxdb</PackageTags>
<PackageIcon>influxdata.jpg</PackageIcon>
<PackageIconUrl>https://raw.githubusercontent.com/influxdata/influxdb-client-csharp/master/Assets/influxdata.jpg</PackageIconUrl>
<PackageProjectUrl>https://github.com/influxdata/influxdb-client-csharp/tree/master/InfluxDB.Client.AspNetCore</PackageProjectUrl>
<PackageLicenseExpression>MIT</PackageLicenseExpression>
<RepositoryUrl>https://github.com/influxdata/influxdb-client-csharp</RepositoryUrl>
<RepositoryType>git</RepositoryType>

<PackageReleaseNotes>https://github.com/influxdata/influxdb-client-csharp/blob/master/CHANGELOG.md</PackageReleaseNotes>
<PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
<RootNamespace>InfluxDB.Client.AspNetCore</RootNamespace>

<AssemblyOriginatorKeyFile>../Keys/Key.snk</AssemblyOriginatorKeyFile>
<SignAssembly>true</SignAssembly>
```